### ISSUE TITLE
dumbing down facets for anaconda

### ISSUE THREAD

**Issue by martingaleh** [PAIR-code/facets#103]
Facets is probably for experienced programmers with extensive python experience.  Is it possible to dumb it down for amateurs?  For example:
1. The installation.  You can't just git clone because it will just copy this big series of folders into whatever directory you type git clone into.  You have to do it from then exact right directory.  Would that be Libs, pkgs, one of a a number of directories anaconda installs into the computer?  The overview sample notebook adds the ./python directory into the pythonpath so it can find generic_Feature_statistic_generator.  Does that mean we copy the entire directory into /python because those using anaconda don't have a python directory.  We could type print (sys.path) but then do we just pick a directory to drag the files into?  
2. nbextension.  jupyter nbextension install facets-dist/ ONLY works if your in the facets-master directory from the git clone in the first place.  By works, I mean it doesn't complain of an error.  I dont' think it actually worked because when I then installed nbextensions config, facets isn't one of the nbextensions.  
3. embrace conda install protobuf.  I don't know why we're messing with wheels or pip3 (which doesn't exist with anaconda users) or the right environment.  None of that works - who knows what they did to my computer.  Conda didn't work either, but conda install tensorflow worked so maybe that means protobuf did too

**Comment by jameswex** [comment]
I consider this related to issue #8 as well. You are right that installation should be easier. We do plan on making installation easier for use with anaconda and pip, but I don't have a concrete date by which this will be done.

**Comment by martingaleh** [comment]
Its related, but it's not exactly #8.  #8 contemplates a programming solution, like someone has to make something.  I'm just suggesting that the front page is clearer.  I finally got it to work, so I have a little authority.  For example:
For issue 1, when it says:
Installation
git clone https://github.com/PAIR-code/facets
cd facets

It should also say, whatever you do and however you copy the entire facets series of folders, add 
import sys
sys.path.append(r'C:\facets-master\facets-master\facets_overview\python')
with the appropriate directory in the quotes.  As you can see, ignorant as I am, my directory is in C:\

For issue 2, you have to be in the facets-dist/ directory for the jupyter nb extension install thing to work.  This was not clear.

For issue 3 above, Maybe say, if all else fails, do conda install tensorflow and protobuf will magically be in the right directory.

Maybe a checklist of all the components and which directory it would be in would be nice.  Oh and one more thing.  Again, rookie error, but we're all trying to reach as many people as possible right?  Before you do conda install anything, conda has to be in your path.  It doesn't just magically work like ipconfig.  Maybe suggest a good phrase to google if "conda has to be in your path" doesn't make sense as a phrase.

**Comment by dartdog** [comment]
Yup I commented on issue #8 and came back in hopes that I'd see some progress.. I have TF-gpu up and running in Jupyter so I think that qualifies me as reasonably fluent in installing and configuring Jupyter (within Anaconda),, but can't get this going, for a beginning when I tried to install just now using:

    tom@tomServal:~/Documents/facets$ cd ~
    tom@tomServal:~$ jupyter nbextension install facets-dist/
I got:

    PermissionError: [Errno 13] Permission denied: '/usr/local/share/jupyter'
But it should be respecting the root anaconda3 environment that executes now when I type "python" in the command shell.. instead it is trying to install where jupyter does not exist... FWIW TF install from binary using pip does not have this issue(although I install it to a different "activated" env.

Sure would like to use this tool it looks like it could be a big help..  also any possibility it could just read csv? or directly use a pandas DF? I'm pretty sure I can handle the PD.DF> proto stuff.. but....
